### PR TITLE
Add BrowserContextOptions

### LINKED ISSUE #525: Support Browser Context options
**Is your feature request related to a problem? Please describe.**
I need to enable `ignoreHTTPSErrors` (https://playwright.dev/docs/api/class-browser#browser-new-context-option-ignore-https-errors) to properly test the websocket transport in js-libp2p.

**Describe the solution you'd like**
We can probably add these options to the runner and have it passed down to the `browser.newContext` call.

### PR DESCRIPTION
**Pull request by MarcoPolo**
Closes #525

### CODE REVIEW

File: src/runner.js [hunk 1]
```diff
@@ -117,12 +117,14 @@ export class Runner {
...
-      this.context = await pw.launchPersistentContext(
-        this.browserDir,
-        pwOptions
+      this.context = await this.browser.newContext(
+        this.options.browserContextOptions
       )
+    } else {
+      this.context = await pw.launchPersistentContext(this.browserDir, {
+        ...pwOptions,
+        ...this.options.browserContextOptions,
```
Comment (hugomrdias) [inline src/runner.js:126]: probably need to add a default value like `{}` for this setting

**Review by hugomrdias — CHANGES_REQUESTED** [review]

File: src/runner.js (hunk 1 above)
Comment (MarcoPolo) [reply src/runner.js:126]: Do I? https://runkit.com/marcopolo/obj-spread

Comment (hugomrdias) [reply src/runner.js:126]: i would prefer to have it there, for consistency

**Review by hugomrdias — APPROVED** [review]

**Comment by MarcoPolo** [comment]
I'll rebase with signed commits

**Pull request merged**